pybamm/models/submodels/tortuosity/base_tortuosity.py

```diff
@@ -51,4 +51,7 @@ def _get_standard_tortuosity_variables(
             }
             variables.update(leading_order_variables)
 
+        # Override print_name
+        tor.print_name = r"\epsilon^{be}"
```

Comment:
```suggestion
        tor.print_name = r"\epsilon^{b_e}"
```
I think we want the exponent to be `b_e`, can you confirm @tinosulzer?